tools/functional-tester/etcd-tester/checks.go

```diff
@@ -65,6 +66,129 @@ func (hc *hashChecker) Check() (err error) {
 	return nil
 }
 
+type leaseChecker struct {
+	leaseStressers []Stresser
+}
+
+func newLeaseChecker(leaseStressers []Stresser) Checker { return &leaseChecker{leaseStressers} }
+
+func (lc *leaseChecker) Check() error {
+	plog.Info("lease stresser invariant check...")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
what's the point of this `cancel()`? every `invariantCheck` completes; what is being canceled?